src/pages/Onboarding/Onboarding.scss

```diff
@@ -119,41 +127,97 @@
         }
       }
     }
+
+    &#active-account {
+      .image > img {
+        margin-bottom: 10px;
+      }
+    }
   }
 
   footer {
-    margin: 20px 0px;
+    margin: 0 auto;
     bottom: 0px;
     left: 0px;
     right: 0px;
     position: absolute;
     text-align: right;
-    display: flex;
-    flex-direction: row-reverse;
     justify-content: space-between;
     align-items: center;
-
-    .step-dots {
+    
+    .wrapper {
+      padding: 20px 0;
+      width: 90vw;
+      border-top: 2px solid $normal;
+      margin: 0 auto;
       display: flex;
-      flex-direction: row;
-
-      .step-dot {
-        cursor: pointer;
-        width: 0.500em;
-        height: 0.500em;
-        margin: .5ch;
-        background-color: #e0e0e0;
-        border-radius: 50%;
-        display: inline-block;
-
-        &.active {
-          background-color: #f7296e;
+      align-items: center;
+      flex-direction: row-reverse;
+      justify-content: space-between;
+      
+      .step-dots {
+        display: flex;
+        justify-content: center;
+        flex-direction: row;
+
+        .step-dot {
+          cursor: pointer;
+          width: 0.500em;
+          height: 0.500em;
+          margin: .5ch;
+          background-color: $normal;
+          border-radius: 50%;
+          display: inline-block;
+          &.active {
+            background-color: $dramatic;
+          }
         }
       }
     }
+  }
+}
 
-    .start-button {
-      margin-right: 30px;
+@media (max-width:370px) {
+  .onBoarding main {
+    position: static;
+  }
+  section div {
+    flex: 1;
+  }
+  .screenshot-phone,
+  .screenshot-phone2 {
+    transform: scale(.9, .77);
+    padding: 0 !important;
```

Comment:
Have you made a new PR to hide all my review notes? `important` is still in this code, you shouldn't use it.

Reply:
there is nothing to hide and think im hide previous notes, i ref. that this pr will solve older.
https://github.com/aeternity/aepp-identity/pull/138